releasenotes/notes/adapt-vqe-0f71234cb6ec92f8.yaml

```diff
@@ -0,0 +1,27 @@
+---
+features:
+  - |
+    Implements the :class:`qiskit.algorithms.minimum_eigensolvers.AdaptVQE`
+    algorithm. This algorithm uses a
+    :class:`qiskit.algorithms.minimum_eigensolvers.VQE` in combination with a
+    pool of operators from which to build out an
+    :class:`qiskit.circuit.library.EvolvedOperatorAnsatz` adaptively.
```

Comment:
Should we note here that this would be commonly used with the UCC(SD) anstaz from Qiskit Nature which are subclasses of EvolvedOperatorAnsatz. This seems more likely than someone directly using an EvolvedOpAnsatz though it is possible.

Reply:
I added a comment but refrained from trying to place a link, since the Terra docs would not resolve it anyways.

Reply:
Was looking for an additional comment but don't see anything - I was not expecting any link for the reason you state

Reply:
> Was looking for an additional comment but don't see anything - I was not expecting any link for the reason you state

Sorry, I meant comment in the docstring. I should have clarified :smile: 
https://github.com/Qiskit/qiskit-terra/blob/4d254daf0d9c2a5a3ca8b09acc426bea587fbbac/qiskit/algorithms/minimum_eigensolvers/adapt_vqe.py#L54-L55